index.html

```diff
@@ -1282,12 +1278,12 @@
   performance measurements, and software updates.
 </aside>
 
-Different users will want to share different kinds and amounts of [=ancillary data=]
-with websites, including possibly no [=ancillary data=].
+Different [=users=] will want to share different kinds and amounts of
+[=ancillary data=] with [=sites=]. Some [=people=] will not want to share any
+[=ancillary data=] at all.
 
-Aggregation or [=de-identified|de-identification=] of data may make users
-interested in sharing [=ancillary data=] in cases where the user was
-otherwise not interested. These techniques may be especially useful and important
+Users may be encouraged to share [=ancillary data=] if it is aggregated with
```

Comment:
This passive voice is confusing.  I think i understand whats intended, but this reads to me as saying "sites could encourage users to share" when (I think) it means "users might want to share…"

Reply:
```suggestion
Users may be willing to share [=ancillary data=] if it is aggregated with
```